guidelines/terms/21/single-pointer.html

```diff
@@ -1,7 +1,5 @@
 <dt><dfn>single pointer</dfn></dt>
 <dd>
-   	
-   <p>pointer input that operates with one point of contact with the screen, including single taps and clicks, double-taps and clicks, long presses, and path-based gestures</p>
-  				
+   <p>an input that only targets a single point on the page/screen at a time – such as a mouse, stylus, or single finger on a touch screen – as opposed to multi-pointer/multi-touch scenarios</p>
```

Comment:
If "multi-point scenarios" is undesirable - how about 

> ...as opposed to input targeting more than one point at the same time (such as a split tap)


...where some people may not know what a split tap is... sigh

Reply:
went with using "multipoint" and then explaining it a bit further